maros_meszaros/__init__.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+# Copyright 2022 Stéphane Caron
```

Comment:
```suggestion
# Copyright 2023 Inria
```
(to avoid confusion, also nitpicking here since copyright goes to our employer whether written or not)